charts/kargo/templates/external-webhooks-server/cert.yaml

```diff
@@ -13,5 +13,5 @@ spec:
   issuerRef:
     kind: Issuer
     name: kargo-selfsigned-cert-issuer
-  secretName: kargo-external-webhooks-server
+  secretName: kargo-external-webhooks-server-cert
```

Comment:
This is more consistent with how we did things for the API server and the admission control webhooks. Naming this differently also preserves the possibility of using the name `kargo-external-webhooks-server` for a more general purpose repository of external webhook server secrets in the future.